packages/components/scss/bootstrap_imports.scss

```diff
@@ -0,0 +1,61 @@
+/*!
+ * Bootstrap v4.6.2 (https://getbootstrap.com/)
+ * Copyright 2011-2022 The Bootstrap Authors
+ * Copyright 2011-2022 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/main/LICENSE)
+ */
+
+/* Deephaven manually imports only the necessary bootstrap files */
+
+@import 'bootstrap/scss/functions';
+
+// patch the functions
+@import './bootstrap_override_functions.scss';
+
+@import 'bootstrap/scss/variables';
+@import 'bootstrap/scss/mixins';
+
+// patch the mixins
+@import './bootstrap_override_mixins_background_variant.scss';
+@import './bootstrap_override_mixins_buttons.scss';
+@import './bootstrap_override_mixins_forms.scss';
+@import './bootstrap_override_mixins_text_emphasis.scss';
+
+@import 'bootstrap/scss/root';
+@import 'bootstrap/scss/reboot';
+@import 'bootstrap/scss/type';
+// @import 'bootstrap/scss/images';
+@import 'bootstrap/scss/code';
+@import 'bootstrap/scss/grid';
+// @import 'bootstrap/scss/tables';
+
+// NEED THIS THOUGH
```

Comment:
@dsmmcken I copy / pasted this verbatim from your PR. Do you remember any more context for this comment?

Reply:
That was probably just a note to self, a can safely be deleted. Probably that forms was needed and for some  reason I didn't think it was.